android/assets/po/es.po

```diff
@@ -2,860 +2,867 @@
-msgstr  "Seleccionas tu vehículo"
+msgid "Select Your Vehicle"
+msgstr "Selecciona tu vehículo"
 
 #: android/assets/vehicles/2cv.xml:1
 msgctxt "vehicle"
-msgid   "2-Deuch"
-msgstr  "Dos Caballos"
+msgid "2-Deuch"
+msgstr "Dos Caballos"
 
 #: android/assets/vehicles/antonin.xml:1
 msgctxt "vehicle"
-msgid   "Ant On-1"
-msgstr  "Ant On-1"
+msgid "Ant On-1"
+msgstr "Ant On-1"
 
 #: android/assets/vehicles/dark-m.xml:1
 msgctxt "vehicle"
-msgid   "Dark M"
-msgstr  "Dark M"
+msgid "Dark M"
+msgstr "Dark M"
 
 #: android/assets/vehicles/harvester.xml:1
 msgctxt "vehicle"
-msgid   "Harvester"
-msgstr  "Cosechadora"
+msgid "Harvester"
+msgstr "Cosechadora"
 
 #: android/assets/vehicles/jeep.xml:1
 msgctxt "vehicle"
-msgid   "Jeep"
-msgstr  "Jeep"
+msgid "Jeep"
+msgstr "Jeep"
 
 #: android/assets/vehicles/miramar.xml:1
 msgctxt "vehicle"
-msgid   "Miramar"
-msgstr  "Miramar"
+msgid "Miramar"
+msgstr "Miramar"
 
 #: android/assets/vehicles/pickup.xml:1
 msgctxt "vehicle"
-msgid   "Pickup"
-msgstr  "Pickup"
+msgid "Pickup"
+msgstr "Pickup"
 
 #: android/assets/vehicles/police.xml:1
 msgctxt "vehicle"
-msgid   "Highway Patrol"
-msgstr  "Policía"
+msgid "Highway Patrol"
+msgstr "Policía"
 
 #: android/assets/vehicles/red.xml:1
 msgctxt "vehicle"
-msgid   "Red Lightning"
-msgstr  "Rayo Rojo"
+msgid "Red Lightning"
+msgstr "Rayo Rojo"
 
 #: android/assets/vehicles/roadster.xml:1
 msgctxt "vehicle"
-msgid   "Roadster"
-msgstr  "Roadster"
+msgid "Roadster"
+msgstr "Roadster"
 
 #: android/assets/vehicles/rocket.xml:1
 msgctxt "vehicle"
-msgid   "Rocket"
-msgstr  "Cohete"
+msgid "Rocket"
+msgstr "Cohete"
 
 #: android/assets/vehicles/santa.xml:1
 msgctxt "vehicle"
-msgid   "Santa Truck"
-msgstr  "Navidad camión"
+msgid "Santa Truck"
+msgstr "Camión de Santa"
 
 #: android/src/main/com/agateau/pixelwheels/android/AndroidLogExporter.java:58
-msgid   "Share via"
-msgstr  "Compartir con"
+msgid "Share via"
+msgstr "Compartir con"
 
 #: android/src/main/com/agateau/pixelwheels/android/AndroidLogExporter.java:63
-msgid   "Hit a bug? Use this button to report it."
-msgstr  "¿ Has encontrado un error ? Utilizas este botón para reportarlo."
+msgid "Hit a bug? Use this button to report it."
+msgstr "¿Has encontrado un error? Utiliza este botón para notificarlo."
 
 #: android/src/main/com/agateau/pixelwheels/android/AndroidLogExporter.java:68
-msgid   "REPORT BUG"
-msgstr  "REPORTAR UN ERROR"
+msgid "REPORT BUG"
+msgstr "NOTIFICAR DE UN ERROR"
 
 #: core/src/com/agateau/pixelwheels/RewardManagerSetup.java:104
 #: core/src/com/agateau/pixelwheels/RewardManagerSetup.java:128
-msgid   "Hit one vehicle with a missile"
-msgid_plural    "Hit %# vehicles with a missile"
-msgstr[0]       "Golpear uno vehículo con un misil"
-msgstr[1]       "Golpear %# vehículos con un misil"
+msgid "Hit one vehicle with a missile"
+msgid_plural "Hit %# vehicles with a missile"
+msgstr[0] "Golpea un vehículo con un misil"
+msgstr[1] "Golpea %# vehículos con un misil"
```

Comment:
Here the sentence does not use the infinitive form anymore. Should it be the case for the other rewards?

Reply:
Yes, "Leave road one time" should be "Sal de la carretera una vez".